src/topology/separation.lean

```diff
@@ -649,6 +649,10 @@ end
 class t2_space (α : Type u) [topological_space α] : Prop :=
```

Comment:
Does adding `@[mk_iff]` here generate the lemma below for you?

Reply:
It does! I'd not seen that before. Well, it does modulo the fact that I went for alpha implicit and the auto-generated `t2_space_iff` lemma has alpha explicit. What do you advise I do? Just add the tag and delete the lemma? I'd be happy to do that; I'm not too fussed about the binder type, I just want to be able to rewrite.

Reply:
OK I changed the PR to adding this tag. Note that `continuous_def` could probably be done this way, although it would end up being called `continuous_iff` in the root namespace and there are a ton of `metric_space.continuous_iff` etc things.

Reply:
I guess the implicit binder probably is better, although it doesn't make much difference. My main goal was to let you know that `mk_iff` existed!

Reply:
Thanks. I propose leaving things as they are then.